po/vi.po

```diff
@@ -6254,138 +6254,168 @@ msgid ""
 "[`Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html) is the standard "
 "resizable heap-allocated buffer:"
 msgstr ""
+"[`Vec`](https://doc.rust-lang.org/std/vec/struct.Vec.html) là kiểu dữ liệu "
+"chuẩn dạng mảng có thể tự thay đổi kích thước:"
 
 #: src/std-types/vec.md
 msgid "\"v1: len = {}, capacity = {}\""
-msgstr ""
+msgstr "\"v1: độ dài = {}, kích thước = {}\""
 
 #: src/std-types/vec.md
 msgid "\"v2: len = {}, capacity = {}\""
-msgstr ""
+msgstr "\"v2: độ dài = {}, kích thước = {}\""
 
 #: src/std-types/vec.md
 msgid "// Canonical macro to initialize a vector with elements.\n"
-msgstr ""
+msgstr "// Macro để khởi tạo một vector với nhiều phần tử cho trước.\n"
 
 #: src/std-types/vec.md
 msgid "// Retain only the even elements.\n"
-msgstr ""
+msgstr "// Chỉ giữ lại các phần tử chẵn.\n"
 
 #: src/std-types/vec.md
 msgid "\"{v3:?}\""
-msgstr ""
+msgstr "\"{v3:?}\""
 
 #: src/std-types/vec.md
 msgid "// Remove consecutive duplicates.\n"
-msgstr ""
+msgstr "// Loại bỏ các phần tử liên tiếp giống nhau.\n"
 
 #: src/std-types/vec.md
 msgid ""
 "`Vec` implements [`Deref<Target = [T]>`](https://doc.rust-lang.org/std/vec/"
 "struct.Vec.html#deref-methods-%5BT%5D), which means that you can call slice "
 "methods on a `Vec`."
 msgstr ""
+"`Vec` implements [`Deref<Target = [T]>`](https://doc.rust-lang.org/std/vec/"
+"struct.Vec.html#deref-methods-%5BT%5D), nên tất cả các hàm của slice đều có "
+"thể được gọi trên một biến kiểu `Vec`."
 
 #: src/std-types/vec.md
 msgid ""
 "`Vec` is a type of collection, along with `String` and `HashMap`. The data "
 "it contains is stored on the heap. This means the amount of data doesn't "
 "need to be known at compile time. It can grow or shrink at runtime."
 msgstr ""
+"Tương tự `String` và `HashMap`, `Vec` là một kiểu dữ liệu sử dụng để lưu trữ "
+"nhiều phần tử. Dữ liệu được lưu trữ trên bộ nhớ heap, nên không cần phải "
+"biết trước kích thước của dữ liệu tại thời điểm compile. Kích thước của "
+"`Vec` có thể thay đổi tại trong quá trình chạy."
 
 #: src/std-types/vec.md
 msgid ""
 "Notice how `Vec<T>` is a generic type too, but you don't have to specify `T` "
 "explicitly. As always with Rust type inference, the `T` was established "
 "during the first `push` call."
 msgstr ""
+"`Vec<T>` cũng là một kiểu dữ liệu generic, nhưng trong đa số trường hợp ta "
```

Comment:
if you are unsure about generic translation then you could put vietnamese translation in brackets like this: generic type (kiểu dữ liệu chung)?

Reply:
Per https://github.com/google/comprehensive-rust/discussions/2062#discussioncomment-9335645, this comment should be resolved?